### PR TITLE
Internal: Create a hidden Nested Elements HTML experiment [ED-11337]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs-new.js [hunk 1]
```diff
@@ -0,0 +1,552 @@
+import Base from 'elementor-frontend/handlers/base';
```
Comment (hein-obox) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs-new.js:1]: This is a copy of the `nested-tabs.js` file.

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 	setHorizontalScrollAlignment,
 	setHorizontalTitleScrollValues,
-} from 'elementor/assets/dev/js/frontend/utils/flex-horizontal-scroll';
+} from 'elementor-frontend-utils/flex-horizontal-scroll';
+
+export default class NestedTabs extends NestedTabsNew {
+	__construct( settings ) {
+		super.__construct( settings );
+
+		this.isHtmlExperimentActive = elementorFrontendConfig.experimentalFeatures[ 'nested-elements-html' ];
```
Comment (hein-obox) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:12]: The idea is to check per method inside this class if the old or the new version will be used.

All the conditional logic for the experiment will be added in the old file, so that once the experiment will be removed, we only have to replace this handler file with `nested-tabs-new.js`, without having to make any changes.

File: modules/nested-tabs/assets/scss/frontend-old.scss [hunk 3]
```diff
@@ -0,0 +1,372 @@
+//
+// Nested Tabs
+//
+.elementor-widget-n-tabs {
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend-old.scss:4]: This is the original styling with the original selector.

File: core/experiments/manager.php [hunk 4]
```diff
@@ -313,6 +313,12 @@ public function get_feature_option_key( $feature_name ) {
 	}
 
 	private function add_default_features() {
+		$this->add_feature( [
```
Comment (TzviRabinovitch) [inline core/experiments/manager.php:316]: move to nested tabs ?

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
-import Base from 'elementor/assets/dev/js/frontend/handlers/base';
```
Comment (hein-obox) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:1]: Unrelated code fix.

File: core/experiments/manager.php (hunk 4 above)
Comment (hein-obox) [reply core/experiments/manager.php]: New widget created Nested Tabs Html

**Review by TzviRabinovitch — APPROVED** [review]

File: .grunt-config/webpack.js [hunk 6]
```diff
@@ -111,6 +111,7 @@ const entry = {
 	'admin-top-bar': path.resolve( __dirname, '../modules/admin-top-bar/assets/js/admin.js' ),
 	'nested-elements': path.resolve( __dirname, '../modules/nested-elements/assets/js/editor/index.js' ),
 	'nested-tabs': path.resolve( __dirname, '../modules/nested-tabs/assets/js/editor/index.js' ),
+	'nested-tabs-html': path.resolve( __dirname, '../modules/nested-tabs-html/assets/js/editor/index.js' ),
```
Comment (louiswol94) [inline .grunt-config/webpack.js:114]:
```suggestion
	'nested-tabs-v3': path.resolve( __dirname, '../modules/nested-tabs-html/assets/js/editor/index.js' ),
```
Maybe? v2 or v3. 'html' will be confusing in the future.

File: includes/managers/widgets.php [hunk 7]
```diff
@@ -47,6 +48,7 @@ class Widgets_Manager {
 	private $_promoted_widgets = [
 		'nested-elements' => NestedTabs::class,
 		'nested-accordion' => Nested_Accordion::class,
+		'nested-elements-html' => NestedTabsHtml::class,
```
Comment (louiswol94) [inline includes/managers/widgets.php:51]: Same here (and everywhere).

**Review by louiswol94 — CHANGES_REQUESTED** [review]
@TzviRabinovitch @hein-obox In general, how is this going to work... Are we maintaining two widgets now? This is a lot of code.

**Review by louiswol94 — APPROVED** [review]
